simcore/simulation/library/crosslink.cpp

```diff
@@ -61,9 +62,12 @@ void Crosslink::SinglyKMC() {
   kmc_bind.SetPBCs(n_dim_, space_->n_periodic, space_->unit_cell);
 
   /* Calculate probability to bind */
-  double kmc_bind_prob = 0;
+  double kmc_bind_prob = 0;  
```

Comment:
Please remove trailing whitespace. There are linters and fixers that do this automatically in vim (e.g. ALE)